README.md

```diff
@@ -67,149 +55,78 @@ query after doing a search. As you can see, it is pretty simple and understandab
-
-    this.given(/I open the (url|site) "$string"$/, function (type, page, done) {
-        var url = type === 'url' ? page : this.baseUrl + page;
-        this.browser.url(url , done);
-    })
-
-    // ... other predefined snippets
-    // ...
-    // ...
-
-    /**
-     * my login snippet
-     */
-    .given(/^I login as "$string" with password "$string"$/, function(username, password, done) {
-        this.browser
-            .setInput('input#username', username)
-            .setInput('input#password', password)
-            .submitForm('#login')
-            .call(done);
-    })
+```sh
+$ wdio --spec ./test/features/select.feature
 ```
 
-You define your snippet using regular expressions. This is pretty powerful as it allows you to create complex
-sentences with multiple options. Everything that's within `"$string"` gets captured and appended to the
-callback. The last argument is always a callback function that you need to call when your step is done.
-You can access the browser and your WebdriverIO instance with `this.browser`.
-
-To assert values this boilerplate project comes with a [Chai](http://chaijs.com/) integration. Let's say
-you want to check if the username gets display in the header properly. For that we add a new snippet in
 
-__./test/steps/then.js__
-```js
-module.exports = function(dict) {
-
-    this.then(/^I expect that the title is( not)* "$string"$/,
-            require('../support/helper/checkTitle'))
-
-    // ... other predefined snippets
-    // ...
-    // ...
-
-    /**
-     * my check username in header snippet
-     */
-    .then(/^the username "$string" should be present in the header$/, function(username, done) {
-        client.getText('#header .username', function(err, headerUsername) {
-            should.not.exist(err);
-            username.should.equal(headerUsername, 'username in header doesn\'t match');
-        })
-        .call(done);
-    })
-```
+# Using tags
 
-That's it. We created two new snippets to test something on our page. We can use these snippets now
-in our Scenario like this:
+If you want to run only specific tests you can mark your features with tags. These tags will be placed before each feature like so:
 
 ```gherkin
+@Tag
 Feature: ...
-
-Scenario: check if username is present
-    Given I login as "roboter" with password "test123"
-    Then the username "roboter" should be present in the header
 ```
 
-# Comments
-
-You can add additional descriptive comments in your feature files.
-
-```gherkin
-###
-  This is a
-  block comment
-###
-Feature: As a bystander
-    I can watch bottles falling from a wall
-    So that I can be mildly amused
+To run only the tests with specific tag(s) use the `--tags=` parameter like so:
 
-# This is a single line comment
-Scenario: check if username is present
-    Given I login as "roboter" with password "test123"
-    Then the username "roboter" should be present in the header
+```sh
+$ wdio --tags=@Tag,@AnotherTag
```

Comment:
Maybe worth mentioning that people shouldn't put tags on certain features and run only these because wdio would spawn vms for not tagged features too (but would close them immediately)